src/k2/httpproxy/HTTPProxy.cpp

```diff
@@ -378,6 +378,32 @@ seastar::future<nlohmann::json> HTTPProxy::_handleWrite(nlohmann::json&& request
     });
 }
 
+seastar::future<std::tuple<k2::Status,dto::CreateSchemaResponse>> HTTPProxy::_handleCreateSchema(
+    dto::CreateSchemaRequest&& request) {
+    K2LOG_D(k2::log::httpproxy, "Received create schema request {}", request);
```

Comment:
you are already in the k2 namespace. No need for the k2:: specifier in this file

Reply:
Similarly for dto - you have a `using namespace dto` up top. For c++ files, we are ok with "using" clauses so in the file you don't have to write dto::

Reply:
Removed namespace from new code.